wandb/sdk/wandb_run.py

```diff
@@ -1249,9 +1249,6 @@ def _partial_history_callback(
         row = row.copy()
         if row:
             row = self._visualization_hack(row)
-            now = time.time()
-            row["_timestamp"] = row.get("_timestamp", now)
-            row["_runtime"] = row.get("_runtime", now - self._get_start_time())
```

Comment:
where was _runtime used before, do you know?

Reply:
what do you mean?

Reply:
this row["_runtime"] tells (used to tell) you how long since the beginning of a run it's been, right? where was that used?

Reply:
I didn't remove the `_runtime` logic it is just moved to the interface, instead of computing it in the user process the lines I linked above will calculate the new delta time.